### DISCUSSION TITLE
Approval not working

### DISCUSSION THREAD

**Discussion by rafamarra** [ansibleguy76/ansibleforms#153] [Q&A] [answered]
After I upgraded to v4.0.16, the approval task is no longer working.
The job remains running and does not show approval buttons.
In version v4.0.15 it works.
I tried it on newer versions too, like v5.0.0, but it didn't work.

![image](https://github.com/ansibleguy76/ansibleforms/assets/36736374/24c5e84d-b02b-435d-a357-098857d1cf81)

**Comment by ansibleguy76** [comment, marked as answer]
That's correct... It is fixed in 5.0.1 (currently in beta).
Since 4.0.16 there was a major upgrade in the job launch code and a bug was introduced that broke approvals.
